client/app/index.ejs

```diff
@@ -54,16 +54,34 @@
     </head>
     <body data-ng-app="App" data-ng-controller="SessionCtrl" data-ng-cloak class="ng-cloak">
         <div class="col-container">
-            <manager-navbar></manager-navbar>
-            <div class="container-fluid h-100 w-100" style="overflow: auto">
-                <div class="row">
-                    <div data-sidebar-menu class="col-sm-4 col-md-2"></div>
-                    <div class="col-sm-8 col-sm-offset-4 col-md-10 col-md-offset-2 px-0 mt-md-0">
-                        <div class="mainContainer pb-5">
-                            <div data-ui-view></div>
-                        </div>
-                    </div>
+
+            <!-- Skip to main content -->
+            <div class="skipnav">
+                <a class="oui-button oui-button_link" href=""
+                    data-ng-bind="::i18n.common_skip_to_main_content"
+                    data-ng-click="scrollTo('maincontent')">
+                </a>
+            </div>
+            <!-- /Skip to main content -->
+
+            <oui-navbar fixed
+                data-active-link="web"
+                data-brand="::navbar.brand"
+                data-main-links="::navbar.managerLinks"
+                data-aside-links="::navbar.internalLinks">
+            </oui-navbar>
+
+            <div data-sidebar-menu></div>
+
+            <div class="main-wrapper pb-5">
+
+                <!-- Skip content target -->
+                <div class="skiptarget">
+                    <a id="maincontent" href="">-</a>
```

Comment:
Anchor ?

Reply:
Yes, to skip content.
I took this way of doing from here: https://www.w3.org/TR/WCAG20-TECHS/G1.html